src/index.js

```diff
@@ -5,13 +5,16 @@ import { createStore } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import rootReducer from './reducers';
 import App from './components/App';
+import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 
 const store = createStore(rootReducer, composeWithDevTools());
 
 ReactDOM.render(
-  <Provider store={store}>
-    <App />
-  </Provider>,
+  <Router>
```

Comment:
I believe we will want to put <Provider> as the outermost wrapper (around <Router> as well).